receiver/vcenterreceiver/testdata/metrics/integration-metrics.json

```diff
@@ -1,1052 +1,3125 @@
 {
-    "resourceMetrics": [
```

Comment:
Do you mind doing this one too?

Reply:
I copied this from [here](https://github.com/observIQ/opentelemetry-collector-contrib/blob/30a905d74965b7baf0c857367f9c41b7d94ec071/receiver/vcenterreceiver/testdata/metrics/integration-metrics.json). I am not extirely sure what what is required. Can I checkout the main branch file and just add a single entry for both, or is there some criteria around what gets added and where?

Reply:
You can try to run the integration test locally by temporarily removing [these lines](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/024a11cd59dc6bddea1da9e06be16a1b665e8993/receiver/vcenterreceiver/integration_test.go#L15-L16). If that works, they you can use the `WriteMetrics` function to regenerate the whole file.

Otherwise, yeah, you can just start with what's on `main` and manually insert the changes. Ultimately, the goal is that this file represents the data you actually get when the receiver runs against the test container. (ignoring the unpredictable parts)